### DISCUSSION TITLE
Sourcemap for "/@windicss/windi.css" points to missing source files ??

### DISCUSSION THREAD

**Discussion by JenuelDev** [windicss/windicss#891] [Q&A] [answered]
So I was wondering about this, can anyone help me with this? I hope this does not affect the build item. Thanks in advance guys.
![image](https://user-images.githubusercontent.com/31676163/216316333-ee2f5b0f-785f-46db-9f6a-8ba7bba088ec.png)

**Comment by TutorFx** [comment]
same here with Nuxt 3.2.2 && Nitro 2.2.2

**Comment by AndrewBogdanovTSS** [comment]
Same issue

**Comment by maxiosigma** [comment]
'+ WARN

**Comment by Borderliner** [comment]
Same here, very annoying.

**Comment by JenuelDev** [comment, marked as answer]
Going to close this one, it seems this issue does not show up anymore after updating to the latest update.
